flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -89,21 +89,41 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                                     + "seconds suffix, daily expression's formation is startTime-endTime, such as 9:30:30-10:50:20, when exclude from 9:30:30-10:50:20 in Monday and Thursday "
                                     + "we can express it as 9:30:30-10:50:20 && * * * ? * 2,5");
 
-    public static final ConfigOption<Double> TARGET_UTILIZATION =
-            autoScalerConfig("target.utilization")
+    public static final ConfigOption<Double> UTILIZATION_TARGET =
+            autoScalerConfig("utilization.target")
                     .doubleType()
                     .defaultValue(0.7)
-                    .withFallbackKeys(oldOperatorConfigKey("target.utilization"))
+                    .withDeprecatedKeys(autoScalerConfigKey("target.utilization"))
+                    .withFallbackKeys(
+                            oldOperatorConfigKey("utilization.target"),
+                            oldOperatorConfigKey("target.utilization"))
                     .withDescription("Target vertex utilization");
 
-    public static final ConfigOption<Double> TARGET_UTILIZATION_BOUNDARY =
-            autoScalerConfig("target.utilization.boundary")
+    public static final ConfigOption<Double> UTILIZATION_TARGET_BOUNDARY =
+            autoScalerConfig("utilization.target.boundary")
```

Comment:
This option is deprecated, do we need to add a `@Deprecated` annotation here?

Also, IIUC, we don't need to rename the option key name since it's deprecated. It means new users don't need it and we don't maintain it in the future. (Compatibility needs to be considered only in the short term.)

Reply:
If `target.utilization.boundary` will not be deprecated in the future，Is it reasonable?

Reply:
we should not rename this option also, the name you set is incorrect anyways. Keys should not be prefixes of each other (not yaml compliant)